demo/index.html

```diff
@@ -831,27 +831,35 @@
               compact: false,
               pageSizeOptions: [1, 10, 50],
               pageSize: 2,
-              feed: hx.dataTable.objectFeed({
-                headers: [
-                  { name: 'Name', id: 'name', groups: ['Child Group 1', 'Parent Group'] },
-                  { name: 'Age', id: 'age', groups: ['Child Group 1', 'Parent Group'] },
-                  { name: 'Profession', id: 'profession', groups: ['Child Group 2', 'Parent Group'] }
-                ],
-                rows: [
-                  {
-                    id: 0, // hidden details can go here (not in the cells object)
-                    cells: { 'name': 'Bob', 'age': 25, 'profession': 'Developer' }
-                  },
-                  {
-                    id: 1,
-                    cells: { 'name': 'Jan', 'age': 41, 'profession': 'Artist' }
-                  },
-                  {
-                    id: 2,
-                    cells: { 'name': 'Dan', 'age': 41, 'profession': 'Builder' }
-                  }
-                ]
-              })
+              feed: {
+                headers: function (cb) {
+                  cb([
+                    { name: 'Name', id: 'name', groups: ['Child Group 1', 'Parent Group'] },
+                    { name: 'Age', id: 'age', groups: ['Child Group 1', 'Parent Group'] },
+                    { name: 'Profession', id: 'profession', groups: ['Child Group 2', 'Parent Group'] }
+                  ])
+                },
+                rows: function (range, cb) {
+                  cb({
+                    rows: [
+                      {
+                        id: 0, // hidden details can go here (not in the cells object)
+                        cells: { 'name': 'Bob', 'age': 25, 'profession': 'Developer' }
+                      },
+                      {
+                        id: 1,
+                        cells: { 'name': 'Jan', 'age': 41, 'profession': 'Artist' }
+                      },
+                      {
+                        id: 2,
+                        cells: { 'name': 'Dan', 'age': 41, 'profession': 'Builder' }
+                      }
+                    ].slice(range.start, range.end + 1),
+                    filteredCount: range.end >= 3 ? 3 : undefined
```

Comment:
Well, the idea is that the table is able to see that the feed has ended and from thereon displays the final size. I have updated the example to hold on to the fact that the size is known as soon as `range.end >= 3` so that when you go back to only showing the first 2 rows you will see `1-2  of 3` in the paginator